acceptance/workspace/jobs/create-error/out.requests.json

```diff
@@ -0,0 +1,9 @@
+[
```

Comment:
Can we keep outputting json-stream? Individual records without the array.

Reply:
What's the benefit? Does `jq` parse that well?

Reply:
Yes it does. You can try yourself.

It is easier to apply the same transformation to every entry.

Reply:
Missed your comment in https://github.com/databricks/cli/pull/2285#discussion_r1948986795 before creating this PR independently :)

Yeah, it does work as expected, fixing.